compiler/AST/build.cpp

```diff
@@ -319,7 +302,7 @@ Expr* buildDotExpr(BaseAST* base, const char* member) {
     return new CallExpr("chpl_localeID_to_locale",
                         new CallExpr(PRIM_WIDE_GET_LOCALE, base));
   else
-    return new CallExpr(".", base, new_StringSymbol(member));
+    return new CallExpr(".", base, new_CStringSymbol(member));
```

Comment:
I still think it's crazy that we use string literal symbols for this.... but that will have to wait for Mike's work.